test/unit/geometry/EdgesGeometry.js

```diff
@@ -183,7 +183,7 @@ function countEdges ( geom ) {
 
 	if ( geom instanceof THREE.EdgesGeometry ) {
 
-		return geom.getAttribute( 'position' ).length / 2;
+		return geom.getAttribute( 'position' ).count / 2;
```

Comment:
Is `geom.getAttribute()` really needed? And if so, should we use it everywhere, instead of accessing the attribute directly?